### ISSUE TITLE
The "Scale Cluster" button is Enable during the process of deleting the cluster

### ISSUE THREAD

**Issue by olegvn88** [radanalyticsio/oshinko-webui#132]
The "Scale Cluster" button is Enable during the process of deleting the cluster. You can use this button, but cluster anyway will be deleted.

**Preconditions:**
At least one cluster should be created.

**Steps to reproduce:**
1. Open the Oshinko Spark Cluster Management home page.
2. Click on the "Actions" button.
3. Click on the "Delete Cluster" button, and click on the "Delete" button in the opened window.
4. Click again on the "Actions" button and click on the "Scale Cluster" button.
5. Input the Number of workers value equals 2 and click on the "Scale" button.

**Actual result:** The workers count were not scaled and cluster was deleted. 

**Expected result:** The "Scale Cluster" button should be disabled after starting the process of deleting the cluster.

**Comment by crobby** [comment]
I agree here, we should disable the "scale" option unless the cluster is in the "running" state.
